Dockerfile

```diff
@@ -28,4 +28,7 @@ RUN python install.py ${BUILD_ARGS}
 
 WORKDIR ${INSTALLATION_FOLDER}
 
+RUN pip install -r requirements-test.txt
+RUN pip install -e .
```

Comment:
Why install in edit mode here?

Reply:
it's an easy way to run the tests and have correct references. I also think it's fine to have the "main" liibrary coming form the main files